sdk/cosmos/azure-cosmos/azure/cosmos/exceptions.py

```diff
@@ -127,3 +127,12 @@ def _partition_range_is_gone(e):
             and e.sub_status == http_constants.SubStatusCodes.PARTITION_KEY_RANGE_GONE):
         return True
     return False
+
+
+def _container_recreate_exception(e):
+    if (e.status_code == http_constants.StatusCodes.BAD_REQUEST
+            and e.sub_status == http_constants.SubStatusCodes.COLLECTION_RID_MISMATCH)\
+            or (e.status_code == http_constants.StatusCodes.NOT_FOUND)\
```

Comment:
This would trigger on a normal 404? In .NET it seems the combination is different?

https://github.com/Azure/azure-cosmos-dotnet-v3/blob/995f2f8c330168b7cb8870d8a539cd69ea00135b/Microsoft.Azure.Cosmos/src/RenameCollectionAwareClientRetryPolicy.cs#L84-L86

Reply:
This might be an attempt to cover the 404 on Read Throughput operations? But it would trigger on, for example, a document read 404?